ElementX/Sources/Services/Room/RoomSummary/RoomEventStringBuilder.swift

```diff
@@ -24,27 +24,40 @@ struct RoomEventStringBuilder {
         }
         
         switch eventItemProxy.content {
-        case .unableToDecrypt(let encryptedMessage):
-            let errorMessage = switch encryptedMessage {
-            case .megolmV1AesSha2(_, .sentBeforeWeJoined): L10n.commonUnableToDecryptNoAccess
-            case .megolmV1AesSha2(_, .verificationViolation): L10n.commonUnableToDecryptVerificationViolation
-            case .megolmV1AesSha2(_, .unknownDevice), .megolmV1AesSha2(_, .unsignedDevice): L10n.commonUnableToDecryptInsecureDevice
-            default: L10n.commonWaitingForDecryptionKey
+        case .msgLike(let msgLikeContent):
```

Comment:
Lets at least be civilised on our side 😅
```suggestion
        case .msgLike(let messageLikeContent):
```